packages/paste-website/src/pages/index.tsx

```diff
@@ -49,16 +50,34 @@ const IndexPage: React.FC<{}> = (): React.ReactElement => {
         </P>
         <Box marginTop="space80" marginBottom="space80">
           <Box display="inline" marginRight="space40">
-            <Button as="a" href="/getting-started/" size="default" variant="primary">
+            <Button
```

Comment:
😟 We now have `<button />`s that act as links on our homepage. Is there a way we can still use the navigate API and switch out the element back to an anchor `<a />`?

Reply:
@SiTaggart We could revert back to an `<a />`, prevent the default `href` action, and then use navigate in an `onClick`. What do you think of that approach?

Reply:
yeah, just use the `as` prop, add a matching `href` so if I command-click it still loads the url in a new tab, and use the `onClick` to do client side routing for gatsby. Use :allthethings:

Reply:
@SiTaggart done here: https://github.com/twilio-labs/paste/pull/146